utils/ansible/ftest/vars/Rocky8.yml

```diff
@@ -6,39 +6,51 @@ daos_openmpi_module_path: /usr/share/modulefiles/mpi/openmpi-x86_64
 daos_base_deps:
   rpms:
     - bash
+    - capstone
     - gdb
     - glibc
     - jq
     - lbzip2
-    - mercury
+    - numactl
+    - numactl-libs
+    - numatop
     - platform-python
     - python36
     - systemd
     - tree
 
 daos_dev_deps:
   rpms:
+    - argobots-devel
+    - capstone-devel
+    - bzip2-devel
+    - clustershell
     - cmake
     - createrepo
+    - daos-raft-devel
     - dnf-plugins-core
+    - fuse3-devel
+    - gcc-c++
+    - gcc-gfortran
+    - golang
     - libaec-devel
+    - libattr-devel
+    - libcmocka-devel
+    - libevent-devel
+    - libyaml-devel
     - meson
+    - ninja-build
+    - openssl-devel
+    - python3.11
+    - python3.11-devel
     - python3-clustershell
```

Comment:
and possibly python-clustershell.  For regular ftest then it's installed via pip but I do remember there being something special about it where it's also required outside the environment as well so maybe this is needed still.

Reply:
ClusterShell dependencies are indeed needed outside ftest and the build of DAOS.
Thus I would like to keep them.